exercises/fun_formulas.livemd

```diff
@@ -58,7 +58,7 @@ total_items = Utils.random(completed_items..100)
 
 percentage = nil
 
-Utils.feedback(:percentage, [completed_items, total_items, percentage])
+Utils.feedback(:percentage_random, [completed_items, total_items, percentage])
```

Comment:
Changing the feedback name is totally fine here, but is :percentage still used? I think we need to remove the test for :percentage as well.

Reply:
I'll still merge the PR, the :percentage test can be removed separately if it's not used.